src/gen-utils.ts

```diff
@@ -198,6 +198,66 @@ export function createGlowElement(options: TextGlowProps, defaults: TextGlowProp
 	return strXml
 }
 
+/**
+ * Create gradient elements i.e. `a:gsLst` and `a:lin`
+ * @param {Gradient} gradient
+ * @param {string} internalElements
+ * @see http://officeopenxml.com/drwSp-GradFill.php
+ * @returns XML string
+ */
+function createGradientElements(gradient: Gradient, internalElements: string): string {
+	let strXml = '',
+		stops = gradient.stops
+
+	if (!isGradient(gradient)) {
+		console.warn(`This is not a valid gradient:`, gradient)
```

Comment:
This is handled similar to the colors; when no valid color is found, we log a warning in the console, and uses the default color. See `createColorElement` fn.